server/src/main/java/org/apache/cassandra/sidecar/metrics/server/AuthMetrics.java

```diff
@@ -32,17 +33,23 @@ public class AuthMetrics
     private static final String DOMAIN = SERVER_PREFIX + ".Auth";
     public final NamedMetric<Counter> jwtPemRefreshFailures;
     public final NamedMetric<Counter> jwtPemRefreshSuccesses;
+    public final NamedMetric<Timer> authorizationTime;
```

Comment:
maybe we can add a comment like this?
```suggestion
    /**
     * Captures the time to successfully authorize non-cached requests. Cached authorization requests will
     * not be recorded in this metric
     */
    public final NamedMetric<Timer> authorizationTime;
```